release notes/v0.37.0.md

```diff
@@ -0,0 +1,73 @@
+k6 v0.37.0 is here! 🎉 Mainly it contains fixes and ongoing efforts with refactoring, but it's just because we are preparing some incredible things in upcoming releases!
```

Comment:
```suggestion
k6 v0.37.0 is here! 🎉 Mainly it contains fixes and ongoing efforts with refactoring.
```

Reply:
I find it good to build excitement about future important features, but I am missing the what!